config/envs/default.py

```diff
@@ -84,6 +84,7 @@ class DefaultConfig:
 
     # E.G. "EMAIL": "GOV_NOTIFY_ID"
     REPLY_TO_EMAILS_WITH_NOTIFY_ID = {
+        "digitalplanningteam@communities.gov.uk": "73eecbb1-5dbc-4653-8c58-46aa79151210",
```

Comment:
This ID is listed twice so I guess one of them should be removed?

Reply:
You are right. But I've now edited the email direct on notify so there's just one email address, `digitalplanningteam@communities.gov.uk` with the same existing ID `73eecbb1-5dbc-4653-8c58-46aa79151210`, so this PR can be abandoned

Reply:
Also, for future reference, we can have 2 emails in this dict with the same ID, but the actual `reply-to` email that gets set from notify will be what's in notify, not whatever email we have here. So with this PR the actual `reply-to` on emails sent from notify would be `@levellingup.gov.uk`

Reply:
Updated the domain to `communities.gov.uk`, so we prevent adding an extra line.